src/components/menu/menu-item/menu-item.component.ts

```diff
@@ -127,8 +138,15 @@ export class MenuItemComponent implements OnInit, OnDestroy, FocusableOption {
     @HostListener('click', ['$event'])
     @HostListener('keydown.enter', ['$event'])
     _onClick(event: MouseEvent | KeyboardEvent): void {
+        if (event instanceof MouseEvent) {
+            if (event.screenX === 0 && event.screenY === 0) {
+                return;
+            }
+        }
```

Comment:
We should probably combine these if statements

`if (event instanceof MouseEvent && event.screenX === 0 && event.screenY === 0) {
    return;
}`